tests/ui/asm/x86_64/const.rs

```diff
@@ -1,8 +1,8 @@
 // only-x86_64
 // run-pass
```

Comment:
These are called `//@run` in ui-test

Reply:
That was something I meant to ask you about and then totally forgot, thanks.